midp/gfx.js

```diff
@@ -304,6 +424,8 @@ var currentlyFocusedTextEditor;
         // sync yet.
         var ctx = imageData.contextInfo.context;
 
+        // XXX Shouldn't we be able to simply fillRect here?
+
```

Comment:
Possibly! But maybe the canvas context could have some kind of transformation or clipping rect currently applied?